lib/galaxy_test/api/test_histories.py

```diff
@@ -176,6 +176,47 @@ def test_index_case_insensitive_contains_query(self):
             index_response = self._get(f"histories{query}").json()
             assert len(index_response) == 3
 
+    def test_index_advanced_filter(self):
```

Comment:
Thanks for adding a bit of testing!

Reply:
Np, and there is also more testing depending on this API endpoint in the context of the selenium integration tests.